src/BuildScriptGenerator/DotNetCore/DotnetCorePlatform.cs

```diff
@@ -264,13 +264,24 @@ public bool IsEnabledForMultiPlatformBuild(RepositoryContext ctx)
             return installationScriptSnippet;
         }
 
+        /// <inheritdoc/>
         public string ResolveVersion(string versionToResolve)
         {
             var resolvedVersion = GetVersionUsingHierarchicalRules(versionToResolve);
             resolvedVersion = GetMaxSatisfyingVersionAndVerify(resolvedVersion);
             return resolvedVersion;
         }
 
+        /// <inheritdoc/>
+        public IDictionary<string, string> GetToolsToBeSetInPath(
```

Comment:
Could we make the `detectorResult `global in this case and access from `DefaultPlatformsInformationProvider`